### PR TITLE
MM-20913  add client API Client.GetServerBusy to fetch server busy state and timeout.

### PR DESCRIPTION
**Pull request by wiggin77**
#### Summary
This PR adds a client API `Client.GetServerBusy` to fetch server busy state and timeout.

* deprecates existing `Client.GetServerBusyTimeout` which only returns the timeout.
* required for MMCTL https://mattermost.atlassian.net/browse/MM-20913

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-20913

### CODE REVIEW

File: api4/system_test.go [hunk 1]
```diff
@@ -571,19 +571,20 @@ func TestClearServerBusy(t *testing.T) {
 	})
 }
 
-func TestGetServerBusyExpires(t *testing.T) {
+func TestGetServerBusy(t *testing.T) {
```
Comment (jespino) [inline api4/system_test.go]: Looks like you are replacing the test for the other method with the test for this method, shouldn't be tested both?

Comment (wiggin77) [reply api4/system_test.go]: Yes, definitely.  Fixed via https://github.com/mattermost/mattermost-server/pull/13986/commits/a5ae6983749bae483072d352067d022f11b33b9f   Thanks!

**Comment by wiggin77** [comment]
Hi @jespino , can I merge this now?  Thanks!

**Comment by jespino** [comment]
@wiggin77 yes, sorry the delay

**Pull request merged**